docs/usage/configuration/config_documentation.md

```diff
@@ -462,6 +462,21 @@ See the docs [request log format](../administration/request_log.md).
 * `additional_resources`: Only valid for an 'http' listener. A map of
    additional endpoints which should be loaded via dynamic modules.
 
+  _Added in Synapse 1.86.0_
+
+* UNIX socket support:
+  * `path`: A path and filename for a UNIX socket. Ensure it is in a readable/writeable
+     directory. No default
+    * **Note**: Declaring a `path` is not compatible while also having a `port`
+       configuration for the same `listener`
+    * **Note**: `metrics` and `manhole` are not supported as `types` on UNIX socket `listeners`
+    * The `x_forwarded` option will default to true and may be left out.
+    * Other options that would not make sense to use with a UNIX socket, such as 
+    `bind_address[es]` and `tls` will be ignored and can be removed.
```

Comment:
Seems like we might just want to validate that these options aren't set to avoid people being confused about what they need to have configured. And to avoid some misguided reports like "I have `bind_address` and `tls` set but it doesn't seem to be applied"

Reply:
In the near future, I'm hoping to add in some basic 'fall-over' support. In those instances, leaving these in means that if a Unix socket file isn't found where it says it should be, it will start using the Tcp address and related configuration. That is only in the idea phase still, and I need some Opinions on it before I try and move that forward.

That being said, there is some basic validity checking in place for if a `path` and a `port` are declared simultaneously, [here](https://github.com/matrix-org/synapse/blob/99c850f79821e12ad1895b9505f8612752deea52/synapse/config/server.py#L948-L957). In retrospect, these may not be fully sufficient. I will ponder on this, that is a fair concern to raise.

Reply:
Would you like this touched in *this* PR? Or should that be separate as 'scope creep'?

Reply:
If we're not doing anything like that now and don't have easy utilities to add these constraints, we can push to a future iteration :+1: